.github/workflows/unit.yml

```diff
@@ -64,5 +64,16 @@ jobs:
         run: yarn bundlesize
       - name: Run Jest tests
         run: yarn test:ci
+      - name: Upload Jest coverage report
+        uses: actions/upload-artifact@v3
+        with:
+            name: jest-coverage-report
+            path: coverage/
       - name: Run benchmark smoke tests
         run: CHROME_BINARY=${{ steps.setup-chrome.outputs.chrome-path }} BENCHMARK_SMOKE_TEST=1 yarn test:performance
+      - name: Add step summary
+        # 1. Remove leading/trailing "border" lines from output
+        # 2. Wrap file names in backticks
+        # 3. Convert leading whitespace to non-breaking to approximate plaintext output
```

Comment:
This is an impressively gnarly `sed` command, but it's not a big deal since this is only for CI. If we really wanted to improve the code for this, though, we might:

1. Use the `json` reporter
2. Write our own `.js` script to process it and print out markdown
3. (Ideally) apply it to the Karma tests too, since they use Istanbul as well and I'm sure it can be configured to output JSON

That said, that might be over-engineering for this feature. It's just great to see the coverage report in the GitHub UI!